datacapturing/src/androidTest/java/de/cyface/datacapturing/persistence/PersistenceLayerTest.java

```diff
@@ -49,7 +49,7 @@
     /**
      * An object of the class under test. It is setup prior to each test execution.
      */
-    private PersistenceLayer oocut;
+    private PersistenceLayer<CapturingPersistenceBehaviour> oocut;
```

Comment:
CapturingPersistenceBehaviour because in this test workflows are tested which are used during capturing, e.g. lead finished measurements while one is still running, etc.